lib/client/api.go

```diff
@@ -442,7 +443,7 @@ type Config struct {
 
 	// DTAuthnRunCeremony allows tests to override the default device
 	// authentication function.
-	// Defaults to [dtauthn.NewCeremony().Run].
+	// Defaults to [dtauthn.RunCeremony].
```

Comment:
I think the previous comment was correct? Also, I don't think this expression is suitable for a link.

```suggestion
	// Defaults to "dtauthn.NewCeremony().Run()".
```